plugins/modules/win_gpo_link.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021 Sebastian Gruber ,dacoso GmbH All Rights Reserved.
+# SPDX-License-Identifier: GPL-3.0-only
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_gpo_link
+author: Sebastian Gruber (@sgruber94)
+short_description: Link a GroupPolicy to a specific ActiveDirectory OU
+description:
+- The C(win_gpo_link) module can link GroupPolicy to a specific ActiveDirectory OU.
+requirements:
+  - PowerShell Module GroupPolicy (https://docs.microsoft.com/en-us/powershell/module/grouppolicy)
+  - PowerShell Module activedirectory (https://docs.microsoft.com/en-us/powershell/module/activedirectory)
+seealso:
+  - module: community.windows.win_gpo_force
+  - module: community.windows.win_gpo_manage
```

Comment:
These don't exist yet. I would hold off on adding them until the modules themselves are merged. These values are used to generate our docs and if the module does not exist yet then that process will fail.